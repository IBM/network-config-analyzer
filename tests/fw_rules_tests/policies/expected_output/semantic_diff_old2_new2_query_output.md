|query|src_ns|src_pods|dst_ns|dst_pods|connection|
|---|---|---|---|---|---|
|semantic_diff, config1: new2, config2: old2, key: Added connections between persistent peers||||||
||[demo]|[bank-ui]|[demo]|[account-command]|TCP 8080,UDP 9090,SCTP 7070|
|semantic_diff, config1: new2, config2: old2, key: Removed connections between persistent peers||||||
||[demo]|[bank-ui]|[demo]|[account-command]|TCP 8082,UDP 9091|
