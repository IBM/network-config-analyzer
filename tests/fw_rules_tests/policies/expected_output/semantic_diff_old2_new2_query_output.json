[
  {
    "query": "semantic_diff, config1: new2, config2: old2, key: Added connections between persistent peers",
    "rules": [
      {
        "src_ns": [
          "demo"
        ],
        "src_pods": [
          "bank-ui"
        ],
        "dst_ns": [
          "demo"
        ],
        "dst_pods": [
          "account-command"
        ],
        "connection": [
          {
            "Protocol": "TCP",
            "Ports": [
              8080
            ]
          },
          {
            "Protocol": "UDP",
            "Ports": [
              9090
            ]
          },
          {
            "Protocol": "SCTP",
            "Ports": [
              7070
            ]
          }
        ]
      }
    ]
  },
  {
    "query": "semantic_diff, config1: new2, config2: old2, key: Removed connections between persistent peers",
    "rules": [
      {
        "src_ns": [
          "demo"
        ],
        "src_pods": [
          "bank-ui"
        ],
        "dst_ns": [
          "demo"
        ],
        "dst_pods": [
          "account-command"
        ],
        "connection": [
          {
            "Protocol": "TCP",
            "Ports": [
              8082
            ]
          },
          {
            "Protocol": "UDP",
            "Ports": [
              9091
            ]
          }
        ]
      }
    ]
  }
]
